examples/api/filled_step.py

```diff
@@ -61,9 +61,11 @@ def filled_hist(ax, edges, values, bottoms=None, orientation='v',
     values = np.r_[values, values[-1]]
     bottoms = np.r_[bottoms, bottoms[-1]]
     if orientation == 'h':
-        return ax.fill_betweenx(edges, values, bottoms, **kwargs)
+        return ax.fill_betweenx(edges, values, bottoms, left_margin=False,
+                                **kwargs)
     elif orientation == 'v':
-        return ax.fill_between(edges, values, bottoms, **kwargs)
+        return ax.fill_between(edges, values, bottoms, bottom_margin=False,
```

Comment:
This looks like some sort of suggested change that people should make, but it isn't noted in the change list above.